run-tests.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+set -e
+
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+case $1 in
+    integration)
+        python setup.py test --pytest="--ignore tests/unit"
+        ;;
+    *)
+        python setup.py test --pytest="--ignore tests/integration"
+        ;;
+esac
```

Comment:
> So why doesn't this script actually run the docker test suite alltogether?

I wanted scripts that can be used in both environments, continuing the idea that we support both development environments.